Resources/MessageQueue/MQConnection.py

```diff
@@ -0,0 +1,107 @@
+"""
+Abstract class for management of MQ connections
+"""
+
+import time
+
+from DIRAC import gLogger, S_OK, S_ERROR
+from DIRAC.Core.Utilities.DErrno import EMQUKN
+
+__RCSID__ = "$Id$"
+
+class MQConnectionError( Exception ):
+    def __init__(self,*args,**kwargs):
```

Comment:
Do you actually need to implement the constructor if it is just forwarding the parameters ?